CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_zendesk v0.17.0
+
+## Bug Fixes
+- Fixed an issue in the `zendesk__sla_policies` model where tickets that were opened and solved outside of scheduled hours were not being reported. 
```

Comment:
I know we somewhat mention it in the logic changes within relevant models, but can we explicitly callout here that this issue was identified and addressed for requester_wait_time and agent_work_time slas.